### DISCUSSION TITLE
ocr识别pdf整行缺失问题

### DISCUSSION THREAD

**Discussion by Yeolnim** [PaddlePaddle/PaddleOCR#13000] [Q&A]
- 系统环境/System Environment：Windows10
- 版本号/Version：Paddle： paddlepaddle-gpu==2.3.2.post111 PaddleOCR： paddleocr==2.6.1.2
- 问题相关组件/Related components：ch_PP-OCRv4_det_infer,  ch_PP-OCRv4_rec_infer, ch_ppocr_mobile_v2.0_cls_infer
- 完整报错/Complete Error Message：
ocr识别pdf文档时，出现整行缺失的情况，不使用方向分类器结果正常。
[设备租赁合同2.pdf](https://github.com/PaddlePaddle/PaddleOCR/files/13344885/2.pdf)
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/49664111/72690577-4e26-47da-b670-fe71d9103522)

**Comment by ybch14** [comment]
遇到了同样的问题，use_angle_cls=True的时候会有整行的遗漏，use_angle_cls=False才能避免
